.eslintrc.js

```diff
@@ -29,24 +29,10 @@ module.exports = {
     '@typescript-eslint/ban-ts-comment': ['warn'],
     // TODO: change no-console to error out for prod
     'no-console': ['warn'],
-    // TODO: fix copied js code that makes this warn necessary
-    'no-use-before-define': ['warn'],
-    'linebreak-style': 0,
-    //TODO: make requests asynchronous
     'no-await-in-loop': ['error'],
-    // don't require .vue extension when importing
-    'import/extensions': [
-      'error',
-      'always',
-      {
-        js: 'never',
-        ts: 'never',
-        vue: 'never',
-      },
-    ],
     // disallow reassignment of function parameters
     // disallow parameter object manipulation except for specific exclusions
-    'no-param-reassign': [2, { props: false }],
+    'no-param-reassign': ['error', { props: false }],
```

Comment:
trying to be consistent of using string instead of numbers for linter error levels